conans/model/layout.py

```diff
@@ -1,7 +1,7 @@
 import os
 
 from conans.model.new_build_info import NewCppInfo
-
+from pathlib import Path
```

Comment:
I'd say that the ``Folders`` ``xxxx_folder`` are not intended nor documented for users yet. 
If anything we want to replace the ``xxx_folder`` for the path alternatives. But this is not necessary in this PR, as this is intended only for user recipes which is always ``self.xxxx_path``, and not ``self.folders.xxxx_path``.

So maybe leave this file untouched.

Reply:
okay, I left it untouched